cpp/src/example/example.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <ray/api.h>
+#include <iostream>
+
+class FooTest {
```

Comment:
I think we can give it a more meaningful name. e.g., `Counter` or something. So the example is easier to understand.